packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -25,55 +25,45 @@ const TabFormatter = styled.div`
 const TabContent = styled.div`
   margin-top: -1px;
   background-color: ${props => props.theme.colors.white};
-  overflow: auto;
+  //overflow: auto;
   border-top: 1px solid ${props => props.theme.colors.gray4};
 `;
 
 class TabPanel extends React.Component {
   constructor(props) {
     super(props);
-    const child = Array.isArray(this.props.children)
-      ? props.children[0]
-      : props.children;
     this.state = {
-      value: child.props.name,
-      currentContent: child.props.children
+      parentTabChanged: props.tabChanged
```

Comment:
Why is this in state?

Reply:
If the team needs this now I don't think a mainline bump would hurt, even before our `next` branch gets merged. If you have an idea on how to rework this so it's not a major change though, let's do that as soon as we can so the team isn't blocked.